docs/gaia/gaia.rst

```diff
@@ -582,6 +591,297 @@ To perform a logout
   user_schema_2.table1
   ...
 
+2.2. Uploading table to user space
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It is now possible to persist a table in the private user space. The table to be uploaded can be in a url, a file, a job or a astropy Table.
+
+The table is stored into the user private area in the database. Each user has a database schema. The schema name is 'user_<user_login_name>'.
+
+For instance, if a login name is 'joe', the database schema is 'user_joe'.
+
+Your uploaded table can be referenced as 'user_joe.table_name'
+
+
+2.2.1. Uploading table from url to user space
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: python
+
+  >>> from astroquery.gaia import Gaia
+  >>> Gaia.login()
+  >>> url = "http://tapvizier.u-strasbg.fr/TAPVizieR/tap/sync/?REQUEST=doQuery&lang=ADQL&FORMAT=votable&QUERY=select+*+from+TAP_SCHEMA.columns+where+table_name='II/336/apass9'"
```

Comment:
Are users supposed to know how to construct these big URLs?

Reply:
This is an example of an external URL.
Yes, it is supposed the user knows external URLs where a table can be found.
When I created this example, I decided to use an external URL that can provide real data.
If I use something like "http://tapvizier.u-strasbg.fr/TAPVizieR/tap/async/123456789/results/result", probably will not work (because that job does not exist) and the user can think that the example does not work.

Reply:
Alright. It might help to add in the documentation where that URL came from.

Reply:
OK. Doc. updated.